StreetHawk/Classes/Core/Private/SHDeepLinking.m

```diff
@@ -97,60 +97,10 @@ - (BOOL)processDeeplinkingUrl:(NSURL *)deeplinking withPushData:(PushDataForAppl
     }
     else if (command != nil && [command compare:COMMAND_POINTZI_AUTHOR options:NSCaseInsensitiveSearch] == NSOrderedSame) //pointzi author
```

Comment:
better to just pass the string to notification manager if first condition fails, no need to check explicitly for pointzi

Reply:
else {

Reply:
If the deeplinking url is for pointzi, it needs to return YES to notify customer App that the deeplinking is handled; otherwise return NO. 
The link pattern needs to be checked.

Reply:
If customer is using streethawk and pointzi what will happen in this case, will pointzi get 2 messages ?

Reply:
No, we cannot ignoring the return of NO. No means not handling this deeplinkg, so that customer's App will not take action.

Reply:
Pointzi cannot be customer of StreetHawk, otherwise need two APIs and customer integration need to change code.